pom.xml

```diff
@@ -37,7 +37,7 @@
         <artifactId>maven-hpi-plugin</artifactId>
         <extensions>true</extensions>
         <configuration>
-          <disabledTestInjection>true</disabledTestInjection>
+          <disabledTestInjection>false</disabledTestInjection>
```

Comment:
Or just simply delete the whole plugin section, since you are not overriding anything any more.